libs/ballot-interpreter/src/hmpb-rust/ballot_card.rs

```diff
@@ -175,15 +226,11 @@ impl PaperInfo {
             width: (3.0 / 16.0) * pixels_per_inch as SubPixelUnit,
             height: (1.0 / 16.0) * pixels_per_inch as SubPixelUnit,
         };
-        let grid_size = match ballot_paper_size {
-            BallotPaperSize::Letter => Size {
-                width: 34,
-                height: 41,
-            },
-            BallotPaperSize::Legal => Size {
-                width: 34,
-                height: 53,
-            },
+        let columns_per_inch = 4.0;
+        let rows_per_inch = 4.0;
+        let grid_size = Size {
+            width: (columns_per_inch * width) as GridUnit,
+            height: (rows_per_inch * height) as GridUnit - 3,
```

Comment:
Same formula used in VxDesign

Reply:
Maybe put a comment here pointing to the other location and vice versa?